tests/phpunit/bootstrap.php

```diff
@@ -0,0 +1,2 @@
+<?php
```

Comment:
Unless we have something useful to put in the `bootstrap.php` we could use `vendor/autoload.php` instead..